contracts/main/bridge/AsterizmClientUpgradeableTransparency.sol

```diff
@@ -0,0 +1,511 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "asterizmprotocol/contracts/evm/interfaces/IInitializerSender.sol";
+import "asterizmprotocol/contracts/evm/interfaces/IClientReceiverContract.sol";
+import "asterizmprotocol/contracts/evm/base/AsterizmEnv.sol";
+import "asterizmprotocol/contracts/evm/base/AsterizmWithdrawalUpgradeable.sol";
+import "asterizmprotocol/contracts/evm/libs/AddressLib.sol";
+import "asterizmprotocol/contracts/evm/libs/UintLib.sol";
+import "asterizmprotocol/contracts/evm/libs/AsterizmHashLib.sol";
+
+abstract contract AsterizmClientUpgradeableTransparency is IClientReceiverContract, AsterizmEnv, AsterizmWithdrawalUpgradeable {
```

Comment:
Valid point. However, I would like to to keep this contract as is because it was simply a transparency version of already existing and audited AsterizmClientUpgradeable abstract contract from asterizm library. The original AsterizmClientUpgradeable was following UUPS so I had to make this transparency-pattern one.